### PR TITLE
Ryna - LA106.ag_an_NetExp_FAO_R_C_Y

### PR DESCRIPTION
**Pull request by rynacui**
#439

### CODE REVIEW

**Comment by bpbond** [comment]
...and @rynacui sets the all-time fastest record for a chunk translation, I think.  🐎

File: R/zchunk_LA106.ag_an_NetExp_FAO_R_C_Y.R [hunk 1]
```diff
@@ -33,60 +32,85 @@ module_aglu_LA106.ag_an_NetExp_FAO_R_C_Y_DISABLED <- function(command, ...) {
...
-    #
-    #
-    # NOTE: there are 'match' calls in this code. You probably want to use left_join_error_no_match
-    # For more information, see https://github.com/JGCRI/gcamdata/wiki/Name-That-Function
-    # NOTE: This code uses translate_to_full_table
-    # This function can be removed; see https://github.com/JGCRI/gcamdata/wiki/Name-That-Function
-    # NOTE: This code uses vecpaste
-    # This function can be removed; see https://github.com/JGCRI/gcamdata/wiki/Name-That-Function
-    # ===================================================
+    # Combine FAO-GCAM mapping files of primarey agriculture goods and animal products
```
Comment (bpbond) [inline R/zchunk_LA106.ag_an_NetExp_FAO_R_C_Y.R]: "primary"

File: R/zchunk_LB109.ag_an_ALL_R_C_Y.R [hunk 2]
```diff
@@ -20,8 +20,8 @@ module_aglu_LB109.ag_an_ALL_R_C_Y <- function(command, ...) {
               "L103.ag_Prod_Mt_R_C_Y",
               "L105.an_Food_Mt_R_C_Y",
               "L105.an_Prod_Mt_R_C_Y",
-              FILE = "temp-data-inject/L106.ag_NetExp_Mt_R_C_Y",
```
Comment (bpbond) [inline R/zchunk_LB109.ag_an_ALL_R_C_Y.R]: Thanks for taking care of this!

**Comment by bpbond** [comment]
Holding on @kvcalvin approval. In the meantime should I assign you another chunk Ryna?

**Comment by rynacui** [comment]
Yes, please, @bpbond. Although I probably won't work on dsr until next week, but let's have them lined up.

**Pull request merged**